rpc/lib/client/http_client.go

```diff
@@ -202,6 +314,30 @@ func unmarshalResponseBytes(cdc *amino.Codec, responseBytes []byte, result inter
 	return result, nil
 }
 
+func unmarshalResponseBytesArray(cdc *amino.Codec, responseBytes []byte, results []interface{}) ([]interface{}, error) {
+	var err error
+	var responses []types.RPCResponse
+	err = json.Unmarshal(responseBytes, &responses)
+	if err != nil {
+		return nil, errors.Errorf("Error unmarshalling rpc response: %v", err)
```

Comment:
```suggestion
		return nil, errors.Errorf("error unmarshalling rpc response: %v", err)
```

Reply:
Missed all of these somehow, sorry about that - I think I've got all of them now though!

Reply:
No worries and thanks!